wire-ios-data-model/Source/Model/FeatureConfig/Feature.swift

```diff
@@ -218,6 +219,9 @@ public class Feature: ZMManagedObject {
 
         case .conferenceCalling, .fileSharing, .conversationGuestLinks, .classifiedDomains, .digitalSignature, .mls:
             break
+        case .e2ei:
+            /// TODO: in the next PR
```

Comment:
do you have a ticket id to mention here?

Reply:
Yes, I have a ticket. Added to the comment.